.github/workflows/python_tracing.yml

```diff
@@ -0,0 +1,29 @@
+name: Python tracing CI
```

Comment:
add this to the unit-test files outside of the matric though. too many workflows as of now